lib/calculator.rb

```diff
@@ -0,0 +1,11 @@
+class Calculator
+
+  def calculate_total(something_to_calculate)
+    sum = 0
+    something_to_calculate.each { |hash|
+      sum += hash[:Price].to_i
```

Comment:
Could this method be even simpler?
Why are numbers being stored as Strings rather than Integers?